R/main.R

```diff
@@ -361,11 +361,17 @@ main_do_migrate <- function(x) {
 }
 
 
-write_script <- function(path) {
+write_script <- function(path, versioned = FALSE) {
   if (!isTRUE(is_directory(path))) {
     stop("'path' must be a directory")
   }
-  code <- c("#!/usr/bin/env Rscript", "orderly:::main()")
+  if (versioned) {
```

Comment:
This is needed to satisfy stricter checks added to R's QA where in tests, we must be careful to use the same R version as the tests are using if shelling out to a second process